guess_the_number.js

```diff
@@ -0,0 +1,41 @@
+import rl from 'readline-sync'
+
+const theTruth = Math.floor( Math.random() * 100 ) + 1
+
+const isValid = input => {
+  return  ( parseInt( input ) === true ) ||
+          ( parseInt(input) < 101 && parseInt(input) > 0 )
+}
+
+const getInput = () => {
+  let guess
+  do {
+      guess = rl.question( 'A number between 1 and 100 please: ' )
+      if ( guess === 'exit' )
```

Comment:
Consider using explicit braces for all blocks - it makes it much easier for a human reviewer/maintainer (including you!) to parse, should this code need to be updated.  Dangling if/else are also common causes of bugs, and can be easily avoided by always explicitly denoting blocks with braces.